src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -46,7 +46,9 @@ public static void applicationMainStartup() {
   /** Stops this application context. */
   @AfterClass
   public static void applicationMainShutdown() {
-    SpringApplication.exit(Application.getApplicationContext(), () -> 0);
+    if (null != Application.getApplicationContext()) {
```

Comment:
it's a bit unconventional to reverse the condition structure from the "typical order" (a [yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions)). did you have a specific reason for it in this occasion?

Reply:
I have just copied this if statement over from the other ControllerTest classes, as it was missing here. According to git-blame, you were actually implementing the statement in that way, or at least associated with the commit here:
https://github.com/GIScience/ohsome-api/commit/0b6c890e50294255d85532cbe37b1ae49488975f :P

Anyway, I did not think of any specific reasoning to write it in that order back then, if it was me. I just know that it can make sense to use it for string comparisons to avoid a potential nullpointer exception.

Reply:
lol :rofl: 

> I just know that it can make sense to use it for string comparisons to avoid a potential nullpointer exception.

You mean when using `equals`, as in `"expected".equals(potenitallyNullStringVariable)`? yes, then it actually can make sense to reverse it.

Reply:
Yeah exactly. So you want to point out that it does not make sense here and would be better in `non-yoda` ordering? We can change that then yes, here and in the other test classes.

Reply:
I'd say it's marginally better to change to the conventional order in all 3(?) test classes, if that's ok for you.